configuration/src/fileConfiguration.ts

```diff
@@ -1,16 +1,20 @@
 /* Copyright (c) Microsoft Corporation. All Rights Reserved. */
+/* Copyright (c) Microsoft Corporation. All Rights Reserved. */
 
-import { readFile, access, F_OK } from 'fs';
-import { IConfiguration } from './IConfiguration';
-import { getVal } from './getVal';
 import * as azureStorage from 'azure-storage';
+import { access, F_OK, readFile } from 'fs';
+import { getVal } from './getVal';
+import { IConfiguration } from './IConfiguration';
+const fswatcher = require('chokidar');
```

Comment:
pull in the types for this like we do for other things. you shouldn't have to `require` it